presto-main/src/test/java/com/facebook/presto/operator/scalar/sql/TestArraySqlFunctions.java

```diff
@@ -268,6 +273,11 @@ public void testArrayNLeastFrequent()
         assertFunction("ARRAY_LEAST_FREQUENT(ARRAY [DOUBLE '1.0', DOUBLE '2.0', DOUBLE '3.0'], 2)", new ArrayType(DOUBLE), asList(1.0d, 2.0d));
         assertFunction("ARRAY_LEAST_FREQUENT(ARRAY ['abc', 'bc', 'aaa'], 3)", new ArrayType(createVarcharType(3)), ImmutableList.of("aaa", "abc", "bc"));
         assertFunction("ARRAY_LEAST_FREQUENT(ARRAY ['', '', ' '], 1)", new ArrayType(createVarcharType(1)), ImmutableList.of(" "));
+    }
+
+    @Test
+    public void testArrayNLeastFrequentUniqueCase()
```

Comment:
What does unique mean?

Reply:
Made this change breaking it up and making the name less vague